provision/core/nginx/config/nginx-wp-common.conf

```diff
@@ -7,6 +7,14 @@
 #     include      /etc/nginx/nginx-wp-common.conf;
 #
 # See local-nginx-example.conf-sample for a full example
+
+location = /error.html {
+    ssi on;
+    internal;
+    auth_basic off;
+    root /usr/share/nginx/html;
+}
```

Comment:
Aren't we missing the `error_page` setting here with all the error codes we're catching?

Reply:
those are in the main `http` block, these location blocks can't go in that file though

Reply:
I didn't mean the location blocks, but the error_page line that actually goes in the server block

Reply:
Ok, i've missinterpreted here. Resolving